src/Data/Text/Internal/Fusion.hs

```diff
@@ -98,6 +99,33 @@ stream (Text arr off len) = Stream next off (betweenSize (len `shiftR` 2) len)
               _ -> U8.chr4 n0 n1 n2 n3
 {-# INLINE [0] stream #-}
 
+-- | /O(n)/ @'streamLn' t = 'stream' (t <> \'\\n\')@
+streamLn ::
```

Comment:
I know that it's an internal module, but let's add `@since` nevertheless please.

Reply:
2.1.2 right?

Reply:
Yeah, 2.1.2